Gems/ROS2/Code/Include/ROS2/Frame/ROS2FrameComponent.h

```diff
@@ -84,8 +95,11 @@ namespace ROS2
 
         NamespaceConfiguration m_namespaceConfiguration;
         AZStd::string m_frameName = "sensor_frame";
+        AZStd::string m_jointNameString;
+        AZ::Name m_jointName;
 
         bool m_publishTransform = true;
+        bool m_isDynamic = false;
```

Comment:
@danielemorra98 Does this need a comment? Might not be 100% intuitive what "dynamic" means in the context of the ROS2Frame Component. Could also be (maybe more appropriately) added to the ROS2FrameComponent::IsDynamic() method

Reply:
That comes from development, I think it is a Cache Github oversight